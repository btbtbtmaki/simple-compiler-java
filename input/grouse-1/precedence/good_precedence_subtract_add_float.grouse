main {

  print 6.3 - 3.3 + 2.1 nl;
  print 6.1 - 3.0 + 4.3 nl;
  print 9.5 - 6.7 + 6.7 nl;
  print 2.1 - 9.4 + 1.2 nl;
  print 4.5 - 3.0 + 2.2 nl;
  print 9.0 - 2.7 + 7.7 nl;
  print 4.9 - 2.3 + 7.6 nl;
  print 6.3 - 6.6 + 9.9 nl;
  print 6.7 - 6.1 + 7.2 nl;
  print 1.2 - 9.3 + 6.2 nl;
  print 9.9 - 2.3 + 2.0 nl;
  print 1.2 - 7.6 + 6.1 nl;

  print 8.0 + 7.5 - 1.7 nl;
  print 3.8 + 4.9 - 4.9 nl;
  print 6.2 + 5.4 - 9.9 nl;
  print 5.7 + 1.3 - 2.0 nl;
  print 4.2 + 6.9 - 7.8 nl;
  print 7.8 + 2.4 - 8.8 nl;
  print 3.9 + 5.6 - 6.7 nl;
  print 4.4 + 8.0 - 4.4 nl;
  print 4.2 + 1.5 - 5.7 nl;
  print 7.7 + 1.5 - 6.6 nl;
  print 4.6 + 5.1 - 3.6 nl;
  print 6.2 + 7.2 - 3.3 nl;

  print 6.9 - 7.9 + 1.3 - 3.6 nl;
  print 7.4 - 9.9 + 1.0 - 6.6 nl;
  print 7.0 - 9.6 + 9.7 - 6.6 nl;
  print 6.3 - 9.1 + 2.3 - 4.6 nl;
  print 8.6 - 1.9 + 8.5 - 5.6 nl;
  print 7.5 - 9.9 + 2.4 - 4.5 nl;
  print 7.1 - 5.5 + 3.7 - 7.5 nl;
  print 3.1 - 6.1 + 9.7 - 2.9 nl;
  print 8.4 - 3.5 + 1.0 - 5.0 nl;
  print 5.7 - 2.1 + 7.0 - 8.1 nl;
  print 1.8 - 1.9 + 9.4 - 1.5 nl;
  print 9.7 - 8.1 + 3.4 - 3.5 nl;

  print 5.1 + 1.9 - 3.9 + 7.3 nl;
  print 1.5 + 5.5 - 6.7 + 9.8 nl;
  print 8.7 + 2.5 - 9.5 + 7.4 nl;
  print 1.9 + 2.5 - 3.9 + 3.7 nl;
  print 4.0 + 5.2 - 9.9 + 4.3 nl;
  print 1.9 + 3.9 - 3.3 + 5.8 nl;
  print 7.7 + 4.4 - 3.0 + 3.6 nl;
  print 7.5 + 6.9 - 9.3 + 4.0 nl;
  print 8.3 + 8.9 - 6.5 + 6.0 nl;
  print 6.3 + 3.2 - 8.5 + 1.6 nl;
  print 4.9 + 2.4 - 1.1 + 6.9 nl;
  print 4.4 + 6.4 - 7.0 + 5.5 nl;

}

