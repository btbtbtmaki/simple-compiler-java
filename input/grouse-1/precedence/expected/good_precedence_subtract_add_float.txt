5.1
7.4
9.5
-6.1
3.7
14
10.2
9.6
7.8
-1.9
9.6
-0.3
13.8
3.8
1.7
5
3.3
1.4
2.8
8
0
2.6
6.1
10.1
-3.3
-8.1
0.5
-5.1
9.6
-4.5
-2.2
3.8
0.9
2.5
7.8
1.5
10.4
10.1
9.1
4.2
3.6
8.3
12.7
9.1
16.7
2.6
13.1
9.3
